playwright/src/main/java/com/microsoft/playwright/junit/Options.java

```diff
@@ -0,0 +1,126 @@
+package com.microsoft.playwright.junit;
+
+import com.microsoft.playwright.APIRequest;
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.BrowserType;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+public class Options {
+  private String baseUrl;
+  private Path storageState;
+  private Viewport viewport;
+  private String channel;
+  private boolean headless;
+  private String browserName;
+  private BrowserType.LaunchOptions launchOptions;
+  private Browser.NewContextOptions contextOption;
+  private APIRequest.NewContextOptions apiRequestOptions;
+
+  public BrowserType.LaunchOptions getLaunchOptions() {
+    return launchOptions;
+  }
+
+  public Options setLaunchOptions(BrowserType.LaunchOptions launchOptions) {
+    this.launchOptions = launchOptions;
+    return this;
+  }
+
+  public Browser.NewContextOptions getContextOption() {
+    return contextOption;
+  }
+
+  public Options setContextOption(Browser.NewContextOptions contextOption) {
+    this.contextOption = contextOption;
+    return this;
+  }
+
+  public APIRequest.NewContextOptions getApiRequestOptions() {
+    return apiRequestOptions;
+  }
+
+  public Options setApiRequestOptions(APIRequest.NewContextOptions apiRequestOptions) {
+    this.apiRequestOptions = apiRequestOptions;
+    return this;
+  }
+
+  public String getBaseUrl() {
+    return baseUrl;
+  }
+
+  public Options setBaseUrl(String baseUrl) {
+    this.baseUrl = baseUrl;
+    return this;
+  }
+
+  public Path getStorageState() {
+    return storageState;
+  }
+
+  public Options setStorageState(Path storageState) {
```

Comment:
nit: it's named setStorageStatePath in the context options, so it may make sense to stay consistent with that.